cfg/pre-commit-config.yaml

```diff
@@ -93,7 +93,10 @@ repos:
   repo: 'local'
 -
   hooks:
-  - id: 'check-added-large-files'
+  -
+    # Ignore emoji-test.txt because it's only 625 KB and the max is 500 KB.
```

Comment:
Could `emoji-test.txt` change to no longer be 625 KB? If so then this comment would be incorrect, which would be worse than no comment

I'd suggest taking out the comment, since `exclude` is pretty self-explanatory

Reply:
Actually, we might not even need this: https://github.com/econia-labs/emojicoin-dot-fun/pull/5#discussion_r1562790451

Reply:
Gotta work on my comment game...I'll keep "no comments are better than incorrect comments" in mind from now on 😅